x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/__mocks__/_mock_server.ts

```diff
@@ -21,6 +23,13 @@ const assertNever = (): never => {
   throw new Error('Unexpected object');
 };
 
+const mockEsPlugins = {
+  // eslint-disable-next-line
+  getCluster: jest.fn((): any => ({
+    callWithRequest: jest.fn((): any => true), // eslint-disable-line
+  })),
+};
```

Comment:
Just a FYI:

In my PR I have something very similar added:

```ts
  const elasticsearch = {
    getCluster: jest.fn().mockImplementation(() => ({
      callWithRequest: jest.fn(),
    })),
  };
```

But when we consolidate on 1 or 2 for now ;-) you can drop the `any` and just do:

```ts
const mockEsPlugins = {
  getCluster: jest.fn(() => ({
    callWithRequest: jest.fn(() => true),
  })),
};
```

The conversion later on in the types that you have 👍 work out well without having to do `any` casting.

```ts
server.plugins.elasticsearch = (mockEsPlugins as unknown) as ElasticsearchPlugin;
```

Reply:
Haha I stole this after seeing it in your last PR  👀. Sounds like a plan. I'll drop the `anys` and keep your naming. Looks better.